release-notes/Cloud/DCP_change_log.md

```diff
@@ -12,11 +12,12 @@ The dataminer.services platform gets updated continuously. This change log can h
 #### 25 April 2024 - Enhancement - Settings overhaul [ID_39471]
 
 The dataminer.services settings, configurable from the Admin app, have been enhanced with the following improvements:
-- From now on settings have a hierarchical structure where disabled parent settings overwrite their sub/child settings. 
-- Settings are now also configureable on the organization where disabled organization settings overwrite the same setting on all its DataMiner Systems. This also applies for all its sub/child settings.
-- Settings are displayed and managed from a seperate page for the organization and for each DMS.
-- A new setting has been added for Live Sharing (dashboard sharing).
-- In addition to one global Remote Access setting, there are now sub/child settings that apply to remote access to Cube, the User-Defined APIs and the web apps separatly.
+
+- Settings now have a hierarchical structure. Disabled parent settings override their child settings.
+- Settings can now also be configured on organization level. If a setting is disabled on organization level, this overrides this same setting for all the DataMiner Systems of this organization, as well as its child settings.
```

Comment:
from our side we tried to not use 'organization/dms **level**' , but ok if this is better